node/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  "extends": "google",
+  "env": {
+    "node": true,
+  },
+  "parserOptions": {
+    "ecmaVersion": 8,
```

Comment:
I will review this PR assuming ES8.

Reply:
After talking to Myles, it seems targeting ES2016 would be safer. Updated this,